src/styles/cards.scss

```diff
@@ -0,0 +1,88 @@
+.container {
```

Comment:
if the `container` is BEM-block move it to a separate file and rename the current file to `card.scss`